debian/rules

```diff
@@ -17,5 +17,9 @@ override_dh_auto_configure:
 override_dh_strip:
 	dh_strip --dbg-package=eos-updater-dbg
 
+# Don't start the services on install
+override_dh_systemd_start:
+	dh_systemd_start --no-start
```

Comment:
@dbnicholson Why wouldn't we want to start the service on install even on converted systems?